sh/tests/left_test.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+# Unofficial Bash Strict Mode
+set -euo pipefail
+IFS='
+'
+script_dirS=$(cd -P "$(dirname "$BASH_SOURCE")" &>/dev/null && pwd)
+
+FILENAME="../student/left.sh"
+cd left
+
+# True if FILE exists and is a regular file
+if [ -f ${FILENAME} ]; then
+    # FILE exists and it's not empty
+    if [ -s ${FILENAME} ]; then
+        if [[ $(cat $FILENAME | grep echo | wc -l) -ne 0 ]]; then
```

Comment:
Should we add a check for the symbol `<` - without it the solution cannot pass?

Reply:
added